shell/common/persistent_cache.cc

```diff
@@ -159,6 +160,27 @@ void PersistentCache::store(const SkData& key, const SkData& data) {
                        std::move(file_name), std::move(mapping));
 }
 
+#if FLUTTER_RUNTIME_MODE != FLUTTER_RUNTIME_MODE_RELEASE
+void PersistentCache::DumpSkp(const SkData& data) {
+  if (is_read_only_ || !IsValid()) {
+    FML_LOG(ERROR) << "Could not dump SKP from read-only or invalid persistent "
+                      "cache.";
+    return;
+  }
+
+  std::stringstream name_stream;
+  int current_index = skp_index_++;
+  skp_index_ %= kMaxSkpIndex;
+  name_stream << "shader_dump_" << current_index << ".skp";
+  std::string file_name = name_stream.str();
+  FML_LOG(INFO) << "Dumping " << file_name;
```

Comment:
Did you mean to leave this log in here? We don't display this log level by default.